idaes/apps/grid_integration/bidder.py

```diff
@@ -11,6 +11,7 @@
 # for full copyright and license information.
 #################################################################################
 import os
+import logging
```

Comment:
Rather than the base logging package, could you use `import idaes.logger as idaeslog`

`_logger = idaeslog.getLogger(__name__)`

Also note that you do not need to set the logger level here, that is done by the user to control the output.